api/app/signals/apps/dsl/evaluators/in_evaluator.py

```diff
@@ -0,0 +1,54 @@
+from django.contrib.gis import geos
+
+from signals.apps.dsl.evaluators.evaluator import Evaluator
+
+
+class InEvaluator(Evaluator):
+    _TIME_FORMAT = '%H:%M:%S'
```

Comment:
Is not needed if we do not support intervals.

Reply:
good catch, is supported via AND rules